javaparser-core/src/main/java/com/github/javaparser/resolution/declarations/ResolvedTypeParameterDeclaration.java

```diff
@@ -93,12 +93,14 @@ public String toString() {
 
             @Override
             public Optional<ResolvedReferenceTypeDeclaration> containerType() {
-                throw new UnsupportedOperationException();
+                throw new UnsupportedOperationException("containerType method is unsupported in " +
+                        "ResolvedTypeParameterDeclaration class.");
             }
 
             @Override
             public ResolvedReferenceType object() {
-                throw new UnsupportedOperationException();
+                throw new UnsupportedOperationException("object method is unsupported in " +
```

Comment:
Object method is **not supported**
There is no added value to specify the class because it is already in the stacktrace